builder-image-apps/quarkus-vertx/src/main/docker/Dockerfile.native

```diff
@@ -0,0 +1,14 @@
+###
+#
+# docker run -i --rm -p 8080:8080 quarkus/using-vertx
+#
+###
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.2
+RUN microdnf --enablerepo=*debug* install gdb glibc-debuginfo zlib-debuginfo
+WORKDIR /work/
+COPY target/*-runner /work/application
+COPY target/*-runner.debug /work/
+COPY target/quarkus-native-image-source-jar/sources /work/sources
```

Comment:
Why don't you keep the same folder hierarchy?

Reply:
Hmm. Dunno. Gonna take a look...